.travis.yml

```diff
@@ -1,21 +1,18 @@
 language: php
 php:
+  - 7.0
   - 5.6
-  - 5.5
-  - 5.4
 env:
-  - EXT_VERSION=1.3.7 DB_PACKAGE=mongodb-10gen
-  - EXT_VERSION=1.3.7 DB_PACKAGE=mongodb-org
-  - EXT_VERSION=1.5.4 DB_PACKAGE=mongodb-10gen
-  - EXT_VERSION=1.5.4 DB_PACKAGE=mongodb-org
+  - EXT_VERSION=1.1 DB_PACKAGE=mongodb-10gen
+  - EXT_VERSION=1.1 DB_PACKAGE=mongodb-org
 before_install:
   - "sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 7F0CEB10"
   - "echo 'deb http://downloads-distro.mongodb.org/repo/ubuntu-upstart dist 10gen' | sudo tee /etc/apt/sources.list.d/mongodb.list"
   - "sudo apt-get update"
   - "sudo apt-get install ${DB_PACKAGE}"
 before_script:
   - composer self-update
-  - yes '' | pecl install -f mongo-${EXT_VERSION}
+  - yes '' | pecl install -f mongodb-${EXT_VERSION}
```

Comment:
Since you goal is to just make the library php7 compatible, I would not worry about upgrading the mongo ext.